packages/flutter/test/material/switch_list_tile_test.dart

```diff
@@ -603,19 +603,660 @@ void main() {
     );
 
     // Start hovering
-    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
+    final TestGesture gesture = await tester.createGesture(
```

Comment:
This test is good, but only tests a very small part of this PR

Reply:
This test is an existing one and only tests the hover color for the `ListTile` instead of the switch part. And because the `SwitchListTile` already has the `hoverColor` for the tile, as we discussed, we might need to rename this api to `tileHoverColor` then introduce the real `hoverColor` for the switch:)

Please let me know if my understanding is not correct.

Reply:
That's okay yeah, but I saw you added more tests. Thanks!